components/SearchFoodsList/styles.module.css

```diff
@@ -11,6 +11,7 @@
   height: 50px;
   padding-left: 8px;
   margin-top: 15px;
+  background: none;
```

Comment:
input border needs to be black and the placeholder text needs to be black

".container > input::placeholder {"